engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/EnsureCompiledJob.scala

```diff
@@ -96,7 +96,7 @@ class EnsureCompiledJob(protected val files: Iterable[File])
         val cacheInvalidationCommands =
           buildCacheInvalidationCommands(
             changeset,
-            module.getLiteralSource
+            module.getSource.getCharacters
```

Comment:
Great to see `Source.getCharacters()` returns something that is faster than `Rope`.